src/pelz/main.c

```diff
@@ -171,6 +171,8 @@ int main(int argc, char **argv)
       else
       {
         load_usage();
+	if (output_size > 1)
```

Comment:
I don't think the ```if (output_size > 1)``` check is necessary. Using ```free()``` on a NULL is a no-op. It should be safe to call ```free()``` without this if()

Reply:
The if check is removed